DSharpPlus/Entities/Channel/Message/DiscordMessage.cs

```diff
@@ -317,6 +318,18 @@ public IReadOnlyList<DiscordMessageSticker> Stickers
         [JsonIgnore]
         private readonly Lazy<IReadOnlyList<DiscordMessageSticker>> _stickersLazy;
 
+        /// <summary>
+        /// Gets emojis for this message
+        /// </summary>
+        [JsonIgnore]
+        public IReadOnlyList<DiscordEmoji> Emojis
+            => this._emojisLazy.Value;
+
+        [JsonProperty("emojis", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
This shouldn't be marked with `[JsonProperty]` since it isn't sent by discord

Reply:
Ok I'll remove it, I thought it was for something else, I'll just ignore it all then.